Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -199,3 +199,73 @@ the size of the cluster.
     ceph orch set backend ""
     ceph mgr module disable rook
     ```
+
+#### Manual configuration of NFS exports
+
```

Comment:
Thank you for the PR! I thought it was an issue at first read-through; my bad. 

I think this is useful information, and I also think that it is on a fine line into documenting Ceph moreso than Rook. Ideally, our docs should refer to Ceph docs when possible and fill in necessary gaps. 

But Ceph docs do often have lots of gaps... The Octopus version of of the docs has the best manual setup info I could find: https://docs.ceph.com/en/octopus/cephfs/nfs/. And yet, the Ceph docs gloss over the client permissions and don't have an export example. I think your examples of those are good to keep.

Having worked with the NFS python code in Ceph, I also notice something that may prove problematic for you or other users. I really would not recommend exactly copying the `export-N` methodology that is used by the Ceph `nfs` module. The `ceph nfs...` module (also nominally used by the dashboard) will overwrite any settings configured that it isn't aware of, and that could cause a working export to be rendered non-working. My recommendation for the next iteration of this doc would be that users should put any manually-configured items (exports or otherwise) only into the rados config object (`conf-nfs.<cephnfs-name>`) to avoid any unforeseen side effects like I described. This recommendation is something missing from our current docs. To keep it short and noticeable, I would make a callout for it like below.

```
!!! note
    To avoid potential conflict with Ceph or Rook processes, we recommend only 
    adding  custom modifications into the config object at `conf-nfs.<cephnfs-name>`.
```

I think my recommendation that configs be put directly into the config object at `conf-nfs.<CephNFS-Name>` will shorten the steps a bit, which will be good as well. As a final request, I think this is most appropriate in the `Documentation/Storage-Configuration/NFS/nfs-advanced.md` file. We can link directly to this advanced topic from `Documentation/Storage-Configuration/NFS/nfs.md` at the end of the (#creating-exports)[Documentation/Storage-Configuration/NFS/nfs.md#creating-exports) section.